en/mkdocs.yml

```diff
@@ -15,8 +15,8 @@
 # under the License.
 
 # Project information
-site_name: Ballerina VS Code Plugin Documentation
-site_description: Ballerina VS Code Plugin Documentation
+site_name: Ballerina VS Code Extension Documentation
+site_description: Ballerina VS Code Extension Documentation
 site_author: WSO2
 site_url: https://wso2.com/ballerina/vscode/docs
 
```

Comment:
Shall we update the edit_uri as given below. The current URL is wrong.

```
edit_uri: https://github.com/wso2/docs-ballerina-visual-studio-code-plugin/tree/main/en/docs
```

Reply:
Done.